android/src/test/java/com/oblador/keychain/cipherStorage/CipherStorageKeystoreAesCbcTests.java

```diff
@@ -0,0 +1,73 @@
+package com.oblador.keychain.cipherStorage;
+
+import android.os.Build;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.oblador.keychain.FakeProvider;
+import com.oblador.keychain.SecurityLevel;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.mockito.junit.VerificationCollector;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.RuntimeEnvironment;
+import org.robolectric.annotation.Config;
+
+import java.security.Key;
+import java.security.Security;
+
+import javax.crypto.SecretKey;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@RunWith(RobolectricTestRunner.class)
+public class CipherStorageKeystoreAesCbcTests {
+  /** Cancel test after 5 seconds. */
```

Comment:
(this is kindof what illustrates the problem with comments that I mentioned)

Reply:
ha ha... nice catch